distributed/client.py

```diff
@@ -3257,6 +3306,9 @@ def get(
                     should_rejoin = False
             try:
                 results = self.gather(packed, asynchronous=asynchronous, direct=direct)
+            except Exception:
+                exc_type, exc_value, exc_traceback = sys.exc_info()
+                raise exc_value.with_traceback(exc_traceback)
```

Comment:
This makes no sense to me.
Are you saying that there was no traceback before?
Also, this code is specific to the sync use case; what happens on async clients?

Reply:
Good catch. I think this is a leftover from the first version of this PR, where I'd only add worker info to exception if any `client` frames were present in traceback; completely unnecessary here.